src/cmds/qsub.c

```diff
@@ -5094,7 +5094,7 @@ daemon_submit(char *qsub_exe, int *do_regular_submit)
 					qsub_exe, fl,
 					h_event, server_out);
 			if (!CreateProcess(NULL, cmd_line, &sa, &sa,
-						TRUE, CREATE_NO_WINDOW, NULL,
+						TRUE, CREATE_NEW_PROCESS_GROUP, NULL,
```

Comment:
I think you still need CREATE_NO_WINDOW also, else it will pop up window

Reply:
Hi Hiren,

I tried CREATE_NEW_PROCESS_GROUP | CREATE_NO_WINDOW as creation flag but it results in python script hanging for 60 secs. 
I too agree that i should add CREATE_NO_WINDOW flag, but combining it with CREATE_NEW_PROCESS_GROUP does not seem to work.

I even tried CREATE_BREAKAWAY_FROM_JOB |  CREATE_NO_WINDOW but even this does not work.

Reply:
try `CREATE_DEFAULT_ERROR_MODE|CREATE_NEW_PROCESS_GROUP` and set `si.lpDesktop = NULL` and make sure you are not getting any pop up windows

Reply:
Hi Hiren, 

Using these flag combination worked for me. But could you please explain little more on si.lpDesktop = NULL. 
I added this before createprocess(..) after referring to other parts of code. Is that the right place?

Reply:
yes, `si.lpDesktop = NULL` is needed only before createprocess and that `si` is not used anywhere else, so don't worry.

Reply:
Thankyou for review & comment. I will rerun all the tests and will update my results.